### PR TITLE
configure Banner to use MDX docs

### PR DESCRIPTION
**Pull request by edburyenegren-okta**
This PR:

- Updates Banner to use MDX docs
- Reorders the Canvas/Docs tabs to prefer Docs
- Adds MDX type declarations to SB
- Deploys to Preview when SB docs are updated

### CODE REVIEW

File: packages/odyssey-storybook/src/components/Banner/Banner.mdx [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+        <OdsIcon icon="caution"></OdsIcon>
+      </span>
+      <p class="ods-banner--content">
+        Severe solar winds detected. Local system flights may be delayed.
+      </p>
+      <section class="ods-banner--actions">
+        <button class="ods-button">View reports</button>
+      </section>
+    </aside>
+  </template>
```
[the remark this answers is not in the document]
Comment (jeffbelser-okta) [reply packages/odyssey-storybook/src/components/Banner/Banner.mdx:98]: got it I'm not opinionated on how you structure the JIRAs, I just don't like seeing all the old HTML + CSS merged back into the repo for a second time now to serve only as a placeholder. Lets keep our commits smaller and more focused by delivering only the markdown needed for the docs in the their current form. We can come back later and add `Actions` or `Anatomy` sections when we have time and do so using the correct examples of our React components

[the remark this answers is not in the document]
Comment (edburyenegren-okta) [reply packages/odyssey-storybook/src/components/Banner/Banner.mdx:98]: In that case, these PRs won't be removing the existing documentation from the archived package, I'll `cp` instead. We need to maintain some convenient copy of what the prior docs looked like before we reach parity.

As discussed, in either event, the result of these initial PRs will **not** be usable or final documentation. I'd ask that people keep that in mind as they review these initial content-migration PRs.

**Comment by edburyenegren-okta** [comment]
This PR can likely be closed, preferring https://github.com/okta/odyssey/pull/1145

Same changes, but applied to all of our components.

File: packages/odyssey-storybook/.storybook/preview.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ export const parameters = {
     },
   },
   previewTabs: {
-    docs: { hidden: true },
+    "storybook/docs/panel": { index: -1 },
```
[the remark this answers is not in the document]
Comment (jeffbelser-okta) [reply packages/odyssey-storybook/.storybook/preview.js:20]: I can see the tab ordering has been changed but I believe there will be additional configuration changes necessary to make the navigation behave the way you're intending here. For example, clicking into your latest storybook build from CI takes you to the `story` pages by default: https://065466a.ods.dev

This behavior is controlled by the URLs that are generated:

story focused URL:
https://065466a.ods.dev/?path=/story/components-banner--info

docs focused URL:
https://065466a.ods.dev/?path=/docs/components-banner--info

File: packages/odyssey-storybook/src/components/Banner/Banner.mdx [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+        </th>
+        <td>Info, Success, Caution, Danger</td>
+        <td>Yes</td>
+        <td>Yes</td>
+        <td>Yes</td>
+        <td>No</td>
+      </tr>
+    </tbody>
+  </table>
+</figure>
```
Comment (jeffbelser-okta) [inline packages/odyssey-storybook/src/components/Banner/Banner.mdx:162]: what's going on here? Do we need this markup as part of this PR?

**Pull request closed without merging**